app/src/main/res/drawable/ic_action_star_list.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
_Reposting the comment in a resolvable thread_

>> Please provide before and after screenshots of the relevant screen parts as suggested in the [contributing guide](https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md).

> Added Screenshots (The after already have some changes to the Size of the Icons that I haven't pushed yet)

Please note the other camera icon usages in the main screen and in the favorites and changes screens.

Reply:
If you do the screenshots in landscape mode then the schedule changes icon also is visible. You can cut out that part - no need to show the whole screen.

Reply:
Thank you for the additional screenshot. - Please update the outdated to reflect the current state of this PR.

Reply:
Done :+1:

Reply:
Can you please try a Pixel 6 emulator? The icons look way smaller there for me.

Reply:
Do you mean all Icons? For me at least, I think most icons have a similar size, except for the List Icons, where the Star/Bell/Bandaid seem to be quite small

Reply:
For me, also the camera icons are smaller on the main screen and in the favorites screen. Likely also in the schedule changes screen, I did not check.

Reply:
I looked at the Screenshots and could not see that big of a difference, for the List Icons it seems that the Smaller Icons were made a bit bigger, which I could try to recreate, for the Camera Icon it might be that the old Icon was filled out entirely while the new Icon has a lot more detail, showing more of the background, which might make it look smaller?

Reply:
Here is how the "no video recording" icon looks like on a Pixel 6 device (old / new):

![no-video-old-new-pixel-6](https://user-images.githubusercontent.com/144518/197629276-cfb250f3-6266-4e2a-aed1-a32e1691aaf6.png)

Reply:
Should be fixed in next push

Reply:
I will check on my device. Why did you also change the XML settings for the bell icon now? Did they also show up small?

Reply:
:heavy_check_mark: Checked here: https://github.com/EventFahrplan/EventFahrplan/pull/518#discussion_r1082605912